.travis.yml

```diff
@@ -24,9 +24,8 @@ matrix:
           sudo: required
           dist: trusty
           group: edge
-        - php: 5.5
-        - php: 5.6
-        - php: 7.0
+        - php: 7.1.0
```

Comment:
deps=low is a per-component build, not a fullstack build. It does not test the requirements of the fullstack